### PR TITLE
Add conda-forge badge to README (address issue #28)

### PR DESCRIPTION
**Pull request by jackirvine97**
This pull request adds a conda-forge badge to the `README.rst`. This badge was created on shields.io as recommended in https://github.com/conda-forge/conda-forge.github.io/issues/258. This addresses issue #28.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -23,7 +23,10 @@ Status
     * - Coverage
       - .. image:: https://codecov.io/gh/brocksam/pyproprop/branch/master/graph/badge.svg
             :target: https://codecov.io/gh/brocksam/pyproprop
-
+    * - conda-forge
```
Comment (brocksam) [inline README.rst:26]: Can we make the "conda-forge" table title read "Anaconda" instead?

**Review by brocksam — COMMENTED** [review]
I think having all the table titles in titlecase looks a bit neater. Also, checkout out `Bokeh <https://github.com/bokeh/bokeh>`_ for some inspiration for how to arranges badges in a README to look way nicer!

File: README.rst (hunk 1 above)
Comment (jackirvine97) [reply README.rst]: Addressed in 3195c0c

**Comment by brocksam** [comment]
Merging but keeping branch so we can reformat the arrangement of the badges to look more pro.

**Pull request merged**